DESCRIPTION

```diff
@@ -29,3 +31,5 @@ Suggests:
     testthat,
     taxize
 RoxygenNote: 6.0.1
+X-schema.org-keywords: species, occurrences, biodiversity, maps
+X-schema.org-applicationCategory: "Data Access"
```

Comment:
hmm... good catch on these not getting into the codemeta.json, that's probably some bug in codemetar we need to fix.  We should take a look at https://github.com/codemeta/codemetar/blob/master/R/codemeta_description.R#L88-L95